Sources/BranchSDK/BranchQRCode.m

```diff
@@ -8,6 +8,7 @@
 #import <LinkPresentation/LPLinkMetadata.h>
 #import "BranchQRCode.h"
 #import "Branch.h"
+#import "BNCServerRequest.h"
```

Comment:
@wpinho-branch Is this extra inclusion ? I dont see any other change in this file.

Reply:
@wpinho-branch https://github.com/BranchMetrics/ios-branch-deep-linking-attribution/actions/runs/19716716237/job/56490475377?pr=1546 - SPM Integration is failing.